Assets/Scripts/KeyBindings/StoreKeyBindings.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+using Newtonsoft.Json;
+
+
+public static class StoreKeyBindings
+{
+    private const string custom_path = "keybindings_custom.json";
+    private const string default_path = "keybindings_default.json";
+
+    static StoreKeyBindings()
+    {
+      
+    }
+        
+    public static void SaveKeyBindings()
+    {
+        Dictionary<String, KeyCode> bindings = KeyBindings.Get();
+
+        var content = JsonConvert.SerializeObject(bindings);
+        File.Delete(custom_path);
```

Comment:
I do not think, that you actually have to remote the file completely.
There should be an overwrite mode.

Reply:
yes it's writeAllText :)